packages/io/amf-deserializer/src/index.js

```diff
@@ -25,6 +25,7 @@ All code released under MIT license
 const version = require('../package.json').version
 const translate = require('./translate')
 const instantiate = require('./deserialize')
+const { ensureString } = require('../../io-utils')
```

Comment:
These IO packages can be installed independently of the full JSCAD tree, so this is probably better as:

`const { ensureString } = require('@jscad/io-utils')`

That makes @jscad/io-utils a dependency for this package. A little pain.

Reply:
Done